### ISSUE TITLE
Fix checkbox column width

### ISSUE THREAD

**Issue by lucasbordeau** [twentyhq/twenty#262]
Reduce checkbox column width & remove right border 

Online: 

![image](https://github.com/twentyhq/twenty/assets/26528466/1651e3fe-f71a-4ef7-a6e8-ec79666086b6)

Figma

![image](https://github.com/twentyhq/twenty/assets/26528466/407c35d6-5731-4312-a698-0d222cc79a09)

**Comment by lucasbordeau** [comment]
I had to make a design decision to solve a bug with the checkboxes that change size. 

Basically, it's because HTML Table automatically resizes all columns when we set it to width: 100%. 

It's the internal algorithm, and we don't have control over it. 

So, to get out of this "automatic mode," we can use the last column to fill the empty space and resize automatically. 

The cool thing with this is that each column keeps its given size no matter what, so it lays the ground for a feature that gives the user a handle to resize columns.
